crescent/ext/tasks/loop.py

```diff
@@ -0,0 +1,37 @@
+from typing import Callable, Sequence
+
+from crescent.ext.tasks.task import TaskCallbackT, _link_task, _Task
+from crescent.internal import MetaStruct
+
+__all__: Sequence[str] = ("loop",)
+
+
+class _Loop(_Task):
```

Comment:
I think these should be public objects (also included in \__all__), mostly for the sake of typehints (and I don't see much a reason to keep them private).